mono/utils/mono-threads.h

```diff
@@ -311,6 +311,13 @@ typedef struct {
 	void (*thread_state_init) (MonoThreadUnwindState *tctx);
 } MonoThreadInfoRuntimeCallbacks;
 
+#define MONO_INIT_THREAD_INFO_RUNTIME_CALLBACKS(prefix) { \
+	prefix ## _setup_async_callback,                  \
+	prefix ## _thread_state_init_from_sigctx,         \
+	prefix ## _thread_state_init_from_handle,         \
+	prefix ## _thread_state_init,                     \
+};
```

Comment:
I don't like these defines since they make looking for methods harder. Can't we just initialize the callbacks normally since it is used with only one prefix ?